examples/NSIDC/.python-version

```diff
@@ -0,0 +1 @@
+miniconda3-latest/envs/nsidc-cloud
```

Comment:
Help me here. What is this? I'm unfamiliar with .python-version files. Naively, this looks like your personal conda environment.

Reply:
Ah hah! I see you add `.python-version` to the `.gitignore`. So I'm now thinking that you did not intend to include this file. Yeah?